libs/journey-bundles/transfer/src/lib/transfer-journey.bundle.ts

```diff
@@ -0,0 +1,60 @@
+import { inject } from '@angular/core';
+import {
+  makeTransferJourney,
+  withConfig,
+  withCommunicationService,
+  MakeTransferJourneyConfiguration,
+  MakeTransferPermissionsService,
+  MakeTransferAccountHttpService,
+  MakeTransferRouteTitleResolverService,
+} from '@backbase/transfer-journey';
+import { JourneyCommunicationService } from '@backbase/shared/feature/communication';
+import { SHARED_JOURNEY_CONFIG } from '@backbase/shared/util/config';
+import { provideJourneyTracker } from '@backbase/foundation-ang/observability';
+import { Routes } from '@angular/router';
+
+// Journey configuration
+const journeyConfig = {
+  maskIndicator: false,
+  maxTransactionAmount: 100,
+  slimMode: false,
+};
+
+// Providers needed by the journey's internal services
+const journeyProviders = [
+  provideJourneyTracker({
+    journeyName: 'transfer',
+  }),
```

Comment:
If something is needed internally in the journey, that should normally be provided in the journeyFactory call.  E.g.:

```ts
export const {
  makeTransferJourney,
  //...
} = journeyFactory({
  journeyName: 'makeTransferJourney',
  //...
  providers: [
    provideJourneyTracker({
      journeyName: 'transfer',
    }),
    MakeTransferAccountHttpService, // even better, this one could be `providedIn: 'root'`
  ]
});
```